src/scripts/model/master.js

```diff
@@ -101,8 +98,8 @@ class MasterModel extends EventEmitter {
         const { opts, parent } = this;
         const { path, enablePolling, interval } = opts;
 
-        if (parent && parent.log instanceof Function) {
-            parent.log(`Fetching data from path: ${path}`);
+        if (opts.log instanceof Function) {
```

Comment:
`.log(...args)` method added to each of M, VM, G5, and component via ctor binding.

When instantiated individually, it is a forwarded call to utils.log. When instantiated by the G5Component top level, it is a shared memsink instance.